zerver/webhooks/discourse/doc.md

```diff
@@ -0,0 +1,21 @@
+On your {{ settings_html|safe }},
+[create a bot](/help/add-a-bot-or-integration) for
+{{ integration_display_name }}. Make sure that you select
+**Incoming webhook** as the **Bot type**:
+
+![](/static/images/help/bot_types.png)
+
+The API keys for "Incoming webhook" bots are limited to only
+sending messages via webhooks. Thus, this bot type lessens
+the security risks associated with exposing the bot's API
+key to third-party services.
+
+Copy the `API KEY` provided - you'll need it later.
+
+Head over to the
+[Discourse Chat Integration Setup Instructions](https://meta.discourse.org/t/68501)
+and complete them.
+
+{!congrats.md!}
+
+![](/static/images/integrations/discourse/001.png)
```

Comment:
We probably want to use the `{!create-stream.md!}` macro in the beginning here. The rest is a partial duplicate of `create-bot-construct-url.md`, but I see why we cannot use that macro here - it contains unfitting instructions.

For the future, maybe we should consider splitting that macro?